src/gazebo_camera_manager_plugin.cpp

```diff
@@ -24,6 +24,8 @@
 #include <boost/algorithm/string.hpp>
 #include <opencv2/opencv.hpp>
 
+// Mavlink includes
+#include <ardupilotmega/ardupilotmega.h>
```

Comment:
This PR is only a 2-liner update. i'll rebase on master showing this after https://github.com/PX4/PX4-SITL_gazebo/pull/868 is merged

Similar update in src/gazebo_gimbal_controller_plugin.cpp file below.